spi/contract-spi/src/main/java/org/eclipse/dataspaceconnector/spi/contract/policy/store/PolicyStore.java

```diff
@@ -0,0 +1,64 @@
+/*
+ *  Copyright (c) 2020 - 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+package org.eclipse.dataspaceconnector.spi.contract.policy.store;
+
+import org.eclipse.dataspaceconnector.policy.model.Policy;
+import org.eclipse.dataspaceconnector.spi.persistence.EdcPersistenceException;
+import org.eclipse.dataspaceconnector.spi.query.QuerySpec;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.stream.Stream;
+
+/**
+ * Persists {@link Policy}.
+ */
+public interface PolicyStore {
```

Comment:
I think this should be in `contract-spi`. We might need access to it in runtimes that don't need the contract stuff.

Reply:
did you mean `:spi:policy-spi`?

Reply:
Makes sense, I must have left it in this module by mistake. I created :spi:policy-spi module. Thanks